package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "novuhq",
   "private": true,
-  "packageManager": "pnpm@8.9.0",
+  "packageManager": "pnpm@9.1.1",
   "scripts": {
     "bootstrap": "npm run setup:dev",
     "start": "npm run jarvis",
     "preinstall": "npx only-allow pnpm",
     "prepare": "husky install",
-    "publish": "lerna publish from-package",
+    "publish": "nx release version && lerna publish from-package",
```

Comment:
Necessary to replace the `workspace:` directives with the actual versions for a successful lerna publish.